server/s3_motr_kvs_writer.cc

```diff
@@ -138,7 +138,8 @@ void S3MotrKVSWriter::create_index_with_oid(
 
   s3_motr_api->motr_idx_init(idx_ctx->idx, &motr_uber_realm, &idx_oid);
   idx_ctx->n_initialized_contexts = 1;
-
+  // Caller will save pv id and other attributes
```

Comment:
In case of global indexes we dont pass this flag right, if so you can mention that